pkgs/top-level/python-packages.nix

```diff
@@ -5569,6 +5571,8 @@ in {
 
   postorius = callPackage ../servers/mail/mailman/postorius.nix { };
 
+  POT = callPackage ../development/python-modules/pot { };
```

Comment:
pname, attr and directory name should match here.

Reply:
Thank you, missed the point about attrname in contribution guide.